dynawo/sources/Common/DYNGraph.cpp

```diff
@@ -130,12 +130,13 @@ Graph::findAllPaths(const unsigned int& vertexOrigin, const unsigned int& vertex
     const unsigned int neighbour = verticesIds_[static_cast<unsigned int>(*neighbourIt)];
     if (encountered.count(neighbour) > 0)
       continue;
+    boost::unordered_set<unsigned int> encounteredIds = encountered;
```

Comment:
this is what causing some paths not to be found ?

Reply:
Yes, because for example if we have the following paths between nodes 6 and 7:
6-1-7 and 6-0-1-7
and if the longer one if found first, then we will not find the shorter one because '1' will be in the set.

Reply:
thanks for the precision